.gitignore

```diff
@@ -2,3 +2,4 @@ target
 Cargo.lock
 !diesel_cli/Cargo.lock
 .env
+.*.sw*
```

Comment:
I'm assuming this is some sort of system specific thing? This should go in your global gitignore, not our project's.